### ISSUE TITLE
Scheduler not working

### ISSUE THREAD

**Issue by lucadiliello** [lucadiliello/transformers-lightning#44]
When using a `TransformersModel` the scheduler on the lr does not work. Maybe the return of `configure_optimizers` should use the `lr_scheduler` key instead of `scheduler`.
